### PR TITLE
Issue#69

### PR DESCRIPTION
**Pull request by AlessandroPaparella**
- Check if there is the "parachain-id" param in service file 
- Check on "relayParentNumber" from "getBlock" rpc method every 30s

### CODE REVIEW

**Comment by Lu191** [comment]
I don't know if checking the `--collator` flag is the right way to identify that a specific node runs a parachain.
For example, from astar [docs](https://docs.astar.network/maintain/node-commands) `--validator` and `--collator` flags seems to behave in the same way

File: srvcheck/chains/substrate.py [hunk 1]
```diff
@@ -36,13 +40,34 @@ def run(self):
...
+		s = Bash('systemctl cat '+service).value()
+		if "--parachain-id" in s:
+			return True
+		return False
+
+	def run(self):
+		if self.prev is None:
+			self.prev = self.chain.getRelayHeight()
+		elif self.prev == self.chain.getRelayHeight():
+			return self.notify(f'Relay is stuck at block {self.prev} {Emoji.Stuck}')
```
Comment (Lu191) [inline srvcheck/chains/substrate.py:61]: Please remove capital letter from `Relay` it should be `relay`

File: srvcheck/chains/substrate.py [hunk 2]
```diff
@@ -36,13 +40,34 @@ def run(self):
...
 
+class TaskRelayChainStuck(Task):
+	def __init__(self, conf, notification, system, chain, checkEvery=30, notifyEvery=60*5):
+		super().__init__('TaskRelayChainStuck',
+			  conf, notification, system, chain, checkEvery, notifyEvery)
+		self.prev = None
+
+	@staticmethod
+	def isPluggable(conf):
+		service = conf.getOrDefault('chain.service')
```
Comment (Lu191) [inline srvcheck/chains/substrate.py:51]: Is not possible to make a function called **`isParachain`**, it should be useful later on, when we'll implement tasks to check block production

**Review by Lu191 — CHANGES_REQUESTED** [review]

**Review by dakk — COMMENTED** [review]
In mangata we have --collator and this will fail; can't we check for both with an or?

You can use queries; since Chain is already instantiated when addTasks is called, we can pass chain to isPluggable.

**Comment by AlessandroPaparella** [comment]
> In mangata we have --collator and this check will fail, right? can't we check for both with an or?
> 
> You can use queries; since Chain is already instantiated when addTasks is called, we can pass chain to isPluggable.

I was thinking that parachain-id was mandatory, so we can check for both collator and validator or make a code refactor in order to add chain to isPluggable...

**Comment by dakk** [comment]
> I was thinking that parachain-id was mandatory, so we can check for both collator and validator or make a code refactor in order to add chain to isPluggable...

I prefer the second solution

**Review by dakk — APPROVED** [review]

**Pull request merged**